rust/crd/src/lib.rs

```diff
@@ -483,10 +484,14 @@ impl ZookeeperCluster {
 
     /// The fully-qualified domain name of the role-level load-balanced Kubernetes `Service`
     pub fn server_role_service_fqdn(&self) -> Option<String> {
+        let cluster_domain = KUBERNETES_CLUSTER_DOMAIN
+            .get()
+            .expect("Could not resolve the Kubernetes cluster domain!");
```

Comment:
I would prefer the longer message
```suggestion
            .expect("KUBERNETES_CLUSTER_DOMAIN must first be set by calling initialize_operator");
```